visual-qontract-services/visual-qontract.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+- hash: ec01dffafc1688617e3d77badf2f09948d10ccfa
```

Comment:
please use `b5f89dec19a42be5e2a664f72a970f7b59ce9acb` because in `ec01dffafc1688617e3d77badf2f09948d10ccfa ` is no os template

Reply:
thanks!